### ISSUE TITLE
Using 2 publishable keys in a Stripe Instance?

### ISSUE THREAD

**Issue by yangtaufoo** [tipsi/tipsi-stripe#308]
Hi All,

I'm using 2 stripe accounts, but i dont find a way to use 2 publishable keys. Is there a way to do it?

Regards

**Comment by cybergrind** [comment]
@yangtaufoo we just init before payment with appropriate creds. Second and furthers inits have not side effects.

**Comment by yangtaufoo** [comment]
alright thanks!

**Comment by smaradona** [comment]
Hi there! I'm having a similar scenario (I need to use two different stripe accounts) and I'm running into an obstacle: I'm using tipsi-stripe version 2.2.0 and I tried doing

`Stripe.init({
  publishableKey: 'Key1',
  merchantId: 'merchantId'
})`

and it's all good the first time I initialize it, but if I try to initialize it for a second time with a different publishable key before making another payment, it just keeps using the first publishable key I used in the first initialize.

Is this the expected behavior? Am I doing something wrong or are further inits only available in the latest versions of tipsi-stripe using `Stripe.setOptions`?

Thanks a lot!
